mkdocs/docs/encodings.md

```diff
@@ -2,153 +2,138 @@
+No compression is applied to the data.
+Depending on the data type, values are stored in the following formats:
 
-- Boolean: Least-significant bit ordering within bytes
-- Integer: Little-Endian byte order
-- Long: Little-Endian byte order
-- Float: IEEE754 floating-point numbers in Little-Endian byte order
-- Double: IEEE754 floating-point numbers in Little-Endian byte order
-- String: length and data streams
+- **Boolean**: Least-significant bit ordering within bytes
+- **Integer**: Little-Endian byte order
+- **Long**: Little-Endian byte order
+- **Float**: IEEE754 floating-point numbers in Little-Endian byte order
+- **Double**: IEEE754 floating-point numbers in Little-Endian byte order
+- **String**: Length and data streams
 
 ## Boolean-RLE
 
-This encoding is used for compressing boolean columns.
-Least-significant bit numbering (bit-endianness) is used as bit order.
-Have a look at the [ORC specification](https://orc.apache.org/specification/ORCv1/#boolean-run-length-encoding)
-for implementation details.
+This encoding compresses boolean columns using least-significant bit numbering (bit-endianness).
+Refer to the [ORC specification](https://orc.apache.org/specification/ORCv1/#boolean-run-length-encoding) for implementation details.
 
 ## Byte-RLE
 
-This encoding is used for compressing byte streams such as the `GeometryType` stream of the Geometry column.
-Have a look at the [ORC specification](https://orc.apache.org/specification/ORCv1/#byte-run-length-encoding)
-for implementation details.
+This encoding compresses byte streams, such as the `GeometryType` stream in Geometry columns.
+Refer to the [ORC specification](https://orc.apache.org/specification/ORCv1/#byte-run-length-encoding) for implementation details.
 
 ## Dictionary Encoding
 
-Dictionary encoding is used to compactly represent repeated values and can be applied to `String` and `Geometry` columns.
-In addition to the actual distinct values stored in the `dictionary` stream, a separate `data` stream for the indices
-into the dictionary is used.
+Dictionary encoding compactly represents repeated values and can be applied to `String` and `Geometry` columns.
+Distinct values are stored in a `dictionary` stream, while a separate `data` stream stores indices into the dictionary.
 
 ### String Dictionary Encoding
 
 The dictionary stream contains distinct UTF-8 encoded string values.
-The data stream contains the indices for the dictionary encoded as `UInt32`.
-A dictionary-encoded nullable string column contains of the following streams and order: `Present`, `Length`, `Dictionary`, `Data`.
+The data stream contains dictionary indices encoded as `UInt32`.
+A dictionary-encoded nullable string column consists of the following streams in order:
+`Present`, `Length`, `Dictionary`, `Data`
 
-All streams can be further compressed by recursively applying lightweight encodings:
+All streams can be further compressed using lightweight encodings:
```

Comment:
Is "can" here correct?
What is the alternative encoding if this is not chosen?
Let's either make this a MUST or specify what is the alternative

Reply:
```suggestion
> [!NOTE]
> TODO: Is "can" (=> MAY) here the right terminoligy?
> This implies that streams might be encoded by one of the options below, but also other options.
> 
> If yes, what is the alternative options?
> If no, clarify this misunderstanding.

All streams can be further compressed using lightweight encodings:
```